elastic/kibana
# [Synthetics] Private locations - no agent flow - add read docs link

## Summary

This adds the read the docs link for the no agent policy flow of Synthetics private locations.

<img width="535" alt="Screen Shot 2022-08-23 at 7 23 02 PM" src="https://user-images.githubusercontent.com/11356435/186282975-903abf59-bbbf-427e-bf29-c1e55d572318.png">

### Testing
1. Start your local Kibana. Ensure that you do not have any agent policy
2. Navigate to monitor management. Click Private locations.
3. You should see the no agent policy prompt. Click read the docs and confirm the link is correct
